ui/base-tool/src/main/java/org/cru/godtools/base/tool/ui/controller/TextController.kt

```diff
@@ -18,6 +18,10 @@ internal class TextController private constructor(
     @AssistedFactory
     interface Factory : BaseController.Factory<TextController>
 
+    init {
+        binding.controller = this
+    }
```

Comment:
Is this needed for overriding ```textIsSelectable```?

Reply:
yes, because in the layout we need to reference the controller now